examples/cfd/vortex_shedding_mgn/inference.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, cfg: DictConfig, logger: PythonLogger):
             models=self.model,
             device=self.device,
         )
-
+        
```

Comment:
Redundant spaces (probably won't pass CI as-is).

Reply:
Thanks! Fixed it - turns out the `black` installation in my env was outdated.